cmd/templ/generatecmd/main.go

```diff
@@ -203,6 +206,14 @@ func processChanges(ctx context.Context, fileNameToLastModTime map[string]time.T
 			return nil
 		}
 		if strings.HasSuffix(path, ".templ") {
+			pathSlices := regexp.MustCompile(`[\\/]`).Split(path, -1)
```

Comment:
Would https://pkg.go.dev/path/filepath#Split be a better choice here? It splits into path and filename, and is aware of the operating system, i.e. Windows vs Unix filesystem path separators.

Even if it wasn't, the regexp compilation would probably be better off outside of the function so that it only happens once per program run.

Reply:
Yeah I think `filepath.Split` would work great here!